.claude/agents/ralph_specialists/infrastructure_specialist.md

```
@@ -0,0 +1,228 @@
+1. Breaking changes to deployed services
+2. Missing infrastructure dependencies
+3. IAM permission gaps
+4. Deployment process compatibility
+
+## Workflow
+
+1. **Receive question** from smart bot
+   Example: "If I modify the auth service, what infrastructure components are affected?"
+
+2. **Examine infrastructure** code
+   ```bash
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)